packages/odyssey-react/src/components/List/List.module.scss

```diff
@@ -11,11 +11,11 @@
  */
 
 .root {
-  max-width: $max-line-length;
-  margin-block-start: 0;
-  margin-block-end: $type-margin;
-  margin-inline: 0;
-  padding-block: 0;
```

Comment:
Quick Q for personal knowledge, do we want to move hardcoded values into the reducer as well or is this just a preference?

Reply:
@francoislehoux-okta only for style declarations that we are currently opening up for theming/customization, not every value! Right now the JIRA's for each component should have a good list of the things we care about but generally, font styles, colors, borders, and margin + padding.